### DISCUSSION TITLE
Welcome to ipylab Discussions!

### DISCUSSION THREAD

**Discussion by jtpio** [jtpio/ipylab#99] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by fleming79** [comment]
Hello,
I've been developing an interface for scientific research and data acquisition for several years using ipywidgets. The code I've developed is intended to run on a local machine with a single user.  Ipylab looks like a good option to enable the widgets to display outside a notebook. 

Below are two screenshots of the interface in action. The first is the widgets inside a notebook.

![image](https://github.com/jtpio/ipylab/assets/1474668/7fe0f84e-2ef2-4300-8bc4-012c9c9f1d77)

The second is using `ipylab` `Panel`. This has enabled to use more of the screen area with the main widget, while adding a log viewer into the right sidebar. The console at the bottom was added using a command linked to a button.

![image](https://github.com/jtpio/ipylab/assets/1474668/10d848ad-ccdd-4a8e-9f78-3c6f1731d3b8)

I'm interested in contributing to Ipylab, I am proficient in Python but not familiar with Javascript.

**Comment by jtpio** [comment]
Thanks @fleming79 for sharing these screenshots, it looks good!

> I'm interested in contributing to Ipylab, I am proficient in Python but not familiar with Javascript.

Thanks! If you would like to contribute don't hesitate to open draft PRs even if they are not finished. That way we can have a look at the JavaScript / TypeScript code more easily if needed.
